src/main/java/io/tiledb/java/api/Dimension.java

```diff
@@ -248,6 +248,38 @@ public FilterList getFilterList() throws TileDBError {
     return this;
   }
 
+  /**
+   * Sets the number of values per cell for the dimension.
+   *
+   * @param cellValNum The number of values per cell
+   * @throws TileDBError
+   */
+  public void setCellValNum(long cellValNum) throws TileDBError {
```

Comment:
Can you add another helper function for `public bool isVarLength()` or `isVar` (whatever you prefer). It should just be something like `return getCellValNum() == TILEDB_VAR_NUM`

Reply:
Just saw that @Shelnutt2. Gonna add this in the next PR